Dockerfile

```diff
@@ -1,10 +1,18 @@
-FROM python:3.4
+FROM python:3.6
 ENV RUNTIME_PACKAGES="python3"
 ENV BUILD_PACKAGES="build-essential python3-dev python3-pip libpq-dev"
 WORKDIR /app
 COPY . /app
 EXPOSE 5050
 RUN apt-get update && apt-get install -y $RUNTIME_PACKAGES $BUILD_PACKAGES
 RUN pip3 install pipenv==8.3.1 && pipenv install --system --deploy
+
+ENV JWT_SECRET testsecret
```

Comment:
Is including environment variables like this in the `Dockerfile` the correct way to supply them? I can't see us doing it in the other repos

Reply:
How do other repos run in docker then?  Are these environment variables in the docker-compose.yml file or does everyone run docker by using `docker run --env env1......`?